doc/StartupGuide.md

```diff
@@ -0,0 +1,155 @@
+    // Add CAMERA event to vocabulary
+    vocabulary.push_back( zeq::EventDescriptor( zeq::hbp::CAMERA,
+					        zeq::hbp::EVENT_CAMERA,
+					        zeq::hbp::SCHEMA_CAMERA, 
+					        zeq::SUBSCRIBER ) );
+    // Publish vocabulary to RESTBridge
+    const zeq::Event& vocEvent = 
+	    zeq::vocabulary::serializeVocabulary( vocabulary );
+    publisher.publish( vocEvent );
+~~~
+    
+## Camera event {#CameraEvent}
+
+The camera event is defined by a 4x4 matrix, with values defined in microns. The
+FlatBuffers definition of the event is the following:
+
+~~~
+    namespace zeq.hbp;
+    table Camera
+    {
+	    matrix:[float];
+    }
+    root_type Camera;
+~~~
+    
+The following figure shows which components of the common 4x4 matrix will be
+changed by which kind of transformation. (T=translation, S=scalation, 
+R{x,y,z}=rotation around the given axis). As you can see, it is safe to set a
+translation and scalation at once, as these share no components.
+
+## ImageJPEG event {#ImageJpegEvent}
+
+The ImageJPEG event is defined by an array of bytes containing a 32bit BGRA
+image. The FlatBuffers definition of the event is the following:
+
+~~~
+    namespace zeq.hbp;
+    table ImageJPEG
+    {
+	    data:[ubyte];
+    }
+    root_type ImageJPEG;
+~~~
+
+Here is a simple example of how to generate and publish the event in the
+rendering resource application code:
+
+~~~
+    #include <zeq/zeq.h>
+    #include <zeq/hbp/hbp.h>
+    const uint8_t* dataPtr = ...; // Buffer containing the JPEG image bytes
+    const uint64_t dataSize = ...; // Buffer size
+    const zeq::hbp::data::ImageJPEG image( dataSize, dataPtr );
+    const zeq::Event& image_event = zeq::hbp::serializeImageJPEG( image );
+    publisher.publish( image_event );
+~~~
+
+## ZeroEQ Handlers {#ZeroEQHandlers}
+
+For every ZeroEQ event supported by the C++ application, a handler must be
+defined and registered. A Handler is a class method that implements the code
+related to the ZeroEQ event. The following example illustrates the registering
+of the MyApplication::onCamera method that takes care of the ZeroEQ Camera
+event.
+
+~~~
+    subscriber_ = new zeq::Subscriber(servus::URI( zeqSchema_ + "cmd://" ));
+    subscriber_->registerHandler( zeq::hbp::EVENT_CAMERA,
+    boost::bind( &MyApplication::onCamera, this , _1 ));
+~~~
+
+## RESTBridge Server {#RESTBridgeServer}
+
+The RestBridge class implements the http server and starts a dedicated thread on
+the host and port specified in the class constructor. HTTP requests addressed to
+the HTTP server is transformed into a ZeroEQ event, according to the registered
+vocabulary. Only GET requests expect a ZeroEQ event in return. If the HTTP 
+request contains a unknown command (not registered in the vocabulary), a 
+BAD_REQUEST (HTTP 400) error is returned.
+
+~~~
+   restbridge::RestBridge mainServer("localhost", 3000);
+   mainServer.run("restSchema");
+~~~
+
```

Comment:
Nice guide, even though it is way more ZeroEQ centric than HTTP/web/REST what should be more emphasized on here imo. Especially, when things are changing in ZeroEQ, this guide will be outdated or needs to be adapted. Some things like the handlers in ZeroEQ are too much detail for me.